README.md

```diff
@@ -1,7 +1,7 @@
 # TTS New Relic Automation
 
 ## Getting Started
-To protect against committing credentials or other secrets:
+To protect agaisnt committing credentials or other secrets:
 - [Gitleaks](https://github.com/zricethezav/gitleaks#getting-started)
 - [Pre-commit](https://pre-commit.com/#install)
 - [Mozilla's Secrets OPerationS (SOPS)](https://github.com/mozilla/sops)
```

Comment:
Is SOPS actually being used for this repository? If not, let's remove from the README to reduce confusion.